app/addons/databases/stores.js

```diff
@@ -112,7 +112,8 @@ const DatabasesStoreConstructor = FauxtonAPI.Store.extend({
     if (!details) {
       return {};
     }
-    const dataSize = details.data_size || details.disk_size || 0;
+    const {sizes} = details;
+    const dataSize = sizes.active || details.file || 0;
```

Comment:
I think `details.file` should be removed here. It doesn't make sense to show different values under the same column.

Reply:
I kept the same "type" of values as it was before but if we want to only stick to one I can change it.